reader/components/highlighter/components/ContextMenu.jsx

```diff
@@ -6,8 +6,11 @@ import { setAnnotation } from '../../../store/document/actions'
 const HIGHLIGHTS_COLORS = ['red', 'yellow', 'green', 'blue']
 
 class ContextMenu extends React.PureComponent {
-  state = {
-    isInnerVisible: true,
+  constructor(props) {
+    super(props)
+    this.state = {
+      isInnerVisible: true,
+    }
```

Comment:
Why did you do this kind of replacements? 

It's really has no sense for me. You do not have dependency between state and props and do not have any business logic in `constructor`.

Reply:
To extend, if you take a look at the numbers, it's _-9 to +13_. I think that you are more productive when you have the opposite, i.e. you remove more code than add.

This assumption come into my mind because product really rarely has less features but if code amount goes down, the team has less to maintain.

Reply:
It is set in the newest version of `eslint-config-airbnb`. I just followed it. Maybe we should disable it. I don't like it either :)